src/arkit.cs

```diff
@@ -536,27 +536,297 @@ interface ARFaceTrackingConfiguration {}
 
 	[iOS (11,0)]
 	[NoWatch, NoTV, NoMac]
+	[StrongDictionary ("ARBlendShapeLocationKeys")]
+	interface ARBlendShapeLocationOptions {
+
+		float BrowDownLeft { get; set; }
+
+		float BrowDownRight { get; set; }
+
+		float BrowInnerUp { get; set; }
+
+		float BrowOuterUpLeft { get; set; }
+
+		float BrowOuterUpRight { get; set; }
+
+		float CheekPuff { get; set; }
+
+		float CheekSquintLeft { get; set; }
+
+		float CheekSquintRight { get; set; }
+
+		float EyeBlinkLeft { get; set; }
+
+		float EyeBlinkRight { get; set; }
+
+		float EyeLookDownLeft { get; set; }
+
+		float EyeLookDownRight { get; set; }
+
+		float EyeLookInLeft { get; set; }
+
+		float EyeLookInRight { get; set; }
+
+		float EyeLookOutLeft { get; set; }
+
+		float EyeLookOutRight { get; set; }
+
+		float EyeLookUpLeft { get; set; }
+
+		float EyeLookUpRight { get; set; }
+
+		float EyeSquintLeft { get; set; }
+
+		float EyeSquintRight { get; set; }
+
+		float EyeWideLeft { get; set; }
+
+		float EyeWideRight { get; set; }
+
+		float JawForward { get; set; }
+
+		float JawLeft { get; set; }
+
+		float JawOpen { get; set; }
+
+		float JawRight { get; set; }
+
+		float MouthClose { get; set; }
+
+		float MouthDimpleLeft { get; set; }
+
+		float MouthDimpleRight { get; set; }
+
+		float MouthFrownLeft { get; set; }
+
+		float MouthFrownRight { get; set; }
+
+		float MouthFunnel { get; set; }
+
+		float MouthLeft { get; set; }
+
+		float MouthLowerDownLeft { get; set; }
+
+		float MouthLowerDownRight { get; set; }
+
+		float MouthPressLeft { get; set; }
+
+		float MouthPressRight { get; set; }
+
+		float MouthPucker { get; set; }
+
+		float MouthRight { get; set; }
+
+		float MouthRollLower { get; set; }
+
+		float MouthRollUpper { get; set; }
+
+		float MouthShrugLower { get; set; }
+
+		float MouthShrugUpper { get; set; }
+
+		float MouthSmileLeft { get; set; }
+
+		float MouthSmileRight { get; set; }
+
+		float MouthStretchLeft { get; set; }
+
+		float MouthStretchRight { get; set; }
+
+		float MouthUpperUpLeft { get; set; }
+
+		float MouthUpperUpRight { get; set; }
+
+		float NoseSneerLeft { get; set; }
+
+		float NoseSneerRight { get; set; }
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[Static]
```

Comment:
Use `[Internal]` so this leaves us the option to expose it as a smart enum later (at the cost of extra manual code or generator changes).

Reply:
Yes good idea, thanks.